inst/css/filter-panel.css

```diff
@@ -357,3 +357,20 @@ a.remove_all:hover {
 .choices_state {
   margin-bottom: .45rem;
 }
+
+.popover-header {
+    padding: .5rem .75rem;
+    margin-bottom: 0;
+    font-size: 1rem;
+    background-color: #f7f7f7;
+    border-bottom: 1px solid #ebebeb;
```

Comment:
I am not sure hard coding colors is the right way to go. Will they not default to theme colors? Nothing much seems to change when I delete these lines.